gateway/tests/test_utils.py

```diff
@@ -122,6 +122,11 @@ def test_get_swagger_url_by_logic_module(self):
         module = factories.LogicModule.create()
         url = get_swagger_url_by_logic_module(module)
         assert url.startswith(module.endpoint)
+    
+    def test_get_swagger_url_by_logic_module_specified_docs(self):
+        module = factories.LogicModule.create(docs_endpoint="api-docs")
+        url = get_swagger_url_by_logic_module(module)
+        assert url.startswith(module.endpoint)
```

Comment:
We need check if the `docs_endpoint` is part of the URL besides `module.endpoint`.